whois-commons/src/main/java/net/ripe/db/whois/common/grs/AuthoritativeResourceImportTask.java

```diff
@@ -65,22 +58,10 @@ public void setEmbeddedValueResolver(final StringValueResolver valueResolver) {
     @Scheduled(cron = "0 15 0 * * *")
```

Comment:
Do you also need to update _AuthoritativeResourceRefreshTask.java_ to poll more frequently? It's currently set to once per hour

Reply:
Grs sources are refreshed on an hourly basis, the main source is refreshed every minute (if there are changes).

Reply:
OK, it's the GRS caches that are refreshed once an hour, thanks.